### DISCUSSION TITLE
blurry ground textures?

### DISCUSSION THREAD

**Discussion by simples12** [kubilus1/autoortho#227] [General]
Hi,

I noticed that my ortho textures look quite blurry sometimes. is this to be expected, or am I doing something wrong?

attached is a screenshot + vram profiler
![sdfdf](https://github.com/kubilus1/autoortho/assets/135391091/05a15dab-c74a-4268-8873-fe17c5124767)
![artjfg](https://github.com/kubilus1/autoortho/assets/135391091/7884815f-e492-43ba-8cc3-9e21645adf87)

**Comment by dpdurst** [comment]
On the lower right, lower center and left to me it does, but that seems to be lower resolution tile that was downloaded. The stuff higher up towards the airport does not like it blurry. The lower right seems to actually be missing a tile as its a different shade of green altogether. Not sure anything you can do that is going to resolve that, Maybe try a high resolution in the config file but then it may or may not work out. Also it may only be this one time download. I don't know if I'd waste a lot of time on this and just enjoy flying

**Reply by Pilotmk3** [reply]
am also having same problem 
here are some screenshots and vram status and xplane12log 
[Log.txt](https://github.com/kubilus1/autoortho/files/13960865/Log.txt)

![Screenshot (297)](https://github.com/kubilus1/autoortho/assets/141163254/1c490a7e-db15-485f-88db-45bcc821355a)
![Screenshot (303)](https://github.com/kubilus1/autoortho/assets/141163254/c00561dc-3fc7-49d6-8f54-6d36d3f11daf)
![Screenshot (304)](https://github.com/kubilus1/autoortho/assets/141163254/ad976b19-d12f-47de-926f-8033fe108066)
![Screenshot (296)](https://github.com/kubilus1/autoortho/assets/141163254/05c55b52-3a41-4a37-98b5-743c0017b65b)
![Screenshot (295)](https://github.com/kubilus1/autoortho/assets/141163254/028b6fe7-deb2-4e5e-9cc5-53a486dbaa17)
